pom.xml

```diff
@@ -113,16 +115,21 @@
 			<artifactId>jackson-databind</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
-                <dependency>
-            		<groupId>com.fasterxml.jackson.core</groupId>
-           		<artifactId>jackson-core</artifactId>
-	            	<version>${jackson.version}</version>
-	        </dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-core</artifactId>
+			<version>${jackson.version}</version>
+		</dependency>
 		<dependency>
 			<groupId>com.fasterxml.jackson.dataformat</groupId>
 			<artifactId>jackson-dataformat-yaml</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.glassfish</groupId>
+			<artifactId>javax.json</artifactId>
```

Comment:
Why ?

Reply:
This is required for Strategic patch
Should I move it to commons module?